tests/ci_build/install/ubuntu_install_nvidia.sh

```diff
@@ -19,13 +19,19 @@
 
 # install nvidia libraries to compile and run CUDA without
 # the necessity of nvidia-docker and a GPU
+set -e
+set -x
 
-# Needed to run add-apt-repository
-apt update && apt install -y software-properties-common
+export DEBIAN_FRONTEND=noninteractive
 
-add-apt-repository -y ppa:graphics-drivers
+mkdir -p /usr/lib/nvidia/
+apt update
+apt install -y --no-install-recommends libcuda1-390
```

Comment:
This package also installs nvidia-390 according to
```
[91m+ apt install -y --no-install-recommends libcuda1-390
The following additional packages will be installed:
  [...] nvidia-390
```

This means that we are back to the old state. Additionally, this does not clearly specify the cuda version.